Added second solution based on size of result set.

james-macrometa opened the pull request:
Updated the import statement to use jsc8@beta. Added second solution and revised copy to cover both solutions.

james-macrometa committed:
initial commit

I have added the Best Practices section to the C8 Query Languages section. I have created placeholder files while I move the articles from Notion.

james-macrometa committed:
new filename and formatting

I also added a missing section from the notion.

james-macrometa committed:
Added second solution based on size of result set.

Updated the import statement to use jsc8@beta. Added second solution and revised copy to cover both solutions.

james-macrometa committed:
Revert "new filename and formatting"

This reverts commit 4abc136876b296c40c16ab5d471870c06cc8e690.

james-macrometa committed:
Revert "initial commit"

This reverts commit 0b4528039cece0ba7553bea201787b244eb6e14b.

diana-macrometa commented on docs/collections/return-data-in-batches.md:
```diff
@@ -0,0 +1,131 @@
+
+## Solutions
+
+There are two recommmended approaches depending on the volume of data being returned. The Cursor API can be used to fetch up to 30,000 records, this is an approximation based on platform limits. Attempting to return more than 30,000 from the Cursor API may cause the query to exceed the query execution time limit and raise an error. The Collection Export API is the recommended approach when retrie ...
```
Please link to the two APIs you are referring to.

diana-macrometa replied:
Generally we want to link to the source on the first usage. I see you link later.

Why are you linking to GitHub rather than API docs?

james-macrometa replied:
The code examples are using the jcs8 SDK so the syntax the user would consult to learn more about the specific methods would be in GH. I see value in including both however, what would be the correct way to add both links without creating confusion?

james-macrometa replied:
Removed the links to the GH docs for now.

diana-macrometa commented on docs/collections/return-data-in-batches.md:
```diff
@@ -0,0 +1,131 @@
+
+There are two recommmended approaches depending on the volume of data being returned. The Cursor API can be used to fetch up to 30,000 records, this is an approximation based on platform limits. Attempting to return more than 30,000 from the Cursor API may cause the query to exceed the query execution time limit and raise an error. The Collection Export API is the recommended approach when retrie ...
+
+Create a connection to the GDN using the jsc8 SDK. You will need an API key, JWT, or Email/Password to complete authentication.
```
```suggestion
Create a connection to the GDN using the jsc8 SDK. You will need an API key, JWT, or Email/Password to complete authentication. For more information, refer to [Authentication](link to authentication topic)
```

diana-macrometa replied:
Notice that you need to add the link path here.

james-macrometa replied:
Added, thank you!

diana-macrometa commented on docs/collections/return-data-in-batches.md:
```diff
@@ -0,0 +1,131 @@
+
+Create a connection to the GDN using the jsc8 SDK. You will need an API key, JWT, or Email/Password to complete authentication.
+
+For the [Cursor API](https://github.com/Macrometacorp/jsC8/blob/master/docs/Reference/Cursor.md#cursor-api) solution, create a cursor to receive the results from your request. When calling the `query` method request options are provided as key/value pairs. It is important to set the `batchSize` value to 1000 and the `stream` value to true. This will permit the results from the query to be returne ...
```
These two paragraphs might make more sense as intro paragraphs under their respective h3s.

james-macrometa replied:
Moved to their respective sections.

james-macrometa committed:
Added API URL links and reformatted document.

Based on recommendations from @diana-macrometa I have added backlinks to the first mentions of the API endpoints and reformatted the document to better position the descriptions of the solutions.

james-macrometa reviewed (commented):
@diana-macrometa please see these changes.

diana-macrometa commented:
If you click the circling arrows next to a person's name, then the system requests another review.

diana-macrometa reviewed (approved):
Looks good! Only thing it needs is a sidebar_position, unless you always want it to be at the very bottom of the sidebar list. Please use a two-digit number. You can check the position by running a local build.

james-macrometa committed:
added sidebar_position

diana-macrometa reviewed (approved):
Looks good! Feel free to merge when ready.

james-macrometa committed:
Merge branch 'master' into james-braswell-docs

james-macrometa committed:
deleted original file after name change.

Removed a duplicated page.

diana-macrometa reviewed (approved):
Once I approve it, you don't need to re-request the review.

james-macrometa commented:
@diana-macrometa I had to remove a file because I had changed the name and it was duplicated. Sorry for the confusion.
